src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,14 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] m = new Machine[3];
```

Comment:
Never use 1-2 letter abbreviations for naming
```suggestion
        Machine[] machines = new Machine[3];
```